tests/acceptance/crates-test.js

```diff
@@ -1,4 +1,4 @@
-import { module, test } from 'qunit';
+yimport { module, test } from 'qunit';
```

Comment:
There's a stray "y" here, causing CI to fail.

Reply:
Oops, thanks for the catch! Pushed a fix.